src/games/strategy/triplea/ui/MapRouteDrawer.java

```diff
@@ -67,11 +67,13 @@ public void drawRoute(final Graphics2D graphics, final RouteDescription routeDes
     final double scale = mapPanel.getScale();
     if (tooFewTerritories || tooFewPoints) {
       if (routeDescription.getEnd() != null) {
-        drawDirectPath(graphics, routeDescription.getStart(), routeDescription.getEnd(), xOffset, yOffset, scale,
-            imageWidth, imageHeight);
+        drawDirectPath(graphics, getPointOnMap(routeDescription.getStart(), xOffset, yOffset, imageWidth, imageHeight),
```

Comment:
It's not clear why the 'getPointOnMap' call is different. Try to see if you can fix up the code so it is more self-documenting. The if/else block here is an indication polymorphism could be used to simplify.

Failing the above, a comment about why we are using the values we are would be of help.